docs/en/engines/table-engines/integrations/mongodb.md

```diff
@@ -6,7 +6,81 @@ sidebar_label: MongoDB
 
 # MongoDB
 
-MongoDB engine is read-only table engine which allows to read data (`SELECT` queries) from remote MongoDB collection. Engine supports only non-nested data types. `INSERT` queries are not supported.
+MongoDB engine is read-only table engine which allows to read data from remote [MongoDB](https://www.mongodb.com/) collection.
+
+Only MongoDB v3.6+ servers are supported.
+
+**If you're facing troubles, please report the issue, and try to use [the legacy implementation](../../../operations/server-configuration-parameters/settings.md#use_legacy_mongodb_integration).
+Keep in mind that it is deprecated, and will be removed in next releases.**
+
+## Types mappings
+
+| MongoDB            | ClickHouse                                                            |
+|--------------------|-----------------------------------------------------------------------|
+| bool, int32, int64 | *any numeric type*, String                                            |
+| int32              | Int32, String                                                         |
+| int64              | Int64, String                                                         |
```

Comment:
i removed int32 and int64 rows.
String because all BSON types can be converted to String. It's important because Mongo does not require strict types, for example, in one document a field can be an int, and in another document be a string